base/src/main/java/com/smartdevicelink/managers/BaseSdlManager.java

```diff
@@ -149,7 +149,17 @@ public void onClosed(LifecycleManager lifeCycleManager, String info, Exception e
 
         @Override
         public void onError(LifecycleManager lifeCycleManager, String info, Exception e) {
+            if (info != null) {
+                DebugTool.logError(TAG, "Error raised: " + info);
+            }
 
+            if (e != null) {
+                DebugTool.logError(TAG, "Exception raised: " + e.getMessage());
+            }
+            
+            if (managerListener != null) {
+                managerListener.onError(info, e);
+            }
```

Comment:
Referring to my comment in `BaseLifecycleManager` can you remove the additions here.

Reply:
@JulianKast Removed (67569c3)